app/app.ts

```diff
@@ -243,4 +254,16 @@ function handleReqDropTable(req: express.Request, res: express.Response) {
 app.get('/drop/:actor_name/:table_name', handleReqDropTable);
 app.get('/drop/:actor_name', handleReqDropTable);
 
+
+app.get('/region/:region/:x/:z', (req: express.Request, res: express.Response) => {
+  const pt = [parseFloat(req.params.x), 0, parseFloat(req.params.z)];
+  if (req.params.region == "Sky") {
+    const indexes = findPolygons(pt, SKY_POLYS, 200);
+    const names = SKY_POLYS.features
+      .filter((feat: any, i: number) => indexes.includes(i))
```

Comment:
nit: a more efficient way to do this would be to map the indices to polygons directly. If n is the number of polygons, doing that would be O(n) instead of O(n^2)

something like this should work:

```ts
const names = findPolygons(pt, SKY_POLYS, 200)
  .map(i => SKY_POLYS.features[i].properties.group);
```

Reply:
Done